jck/jtrunner/JavaTestRunner.java

```diff
@@ -999,6 +1000,11 @@ private static boolean generateSummary() {
 			fw.close();
 			System.out.println(resultSummary.toString());
 			return true; 
+		} catch (FileNotFoundException e) {
+			System.out.println("No XML result file found on disc. Please check if the "
```

Comment:
```suggestion
			System.out.println("No javatest XML result file found. Please check if the "
```

Reply:
The suggested message reads better. Updated!